Autopilot/AttitudeManager/SensorFusion.hpp

```diff
@@ -0,0 +1,33 @@
+/**
+ * Sensor Fusion - Converts raw sensor data into human readable formatted structs
+ * Author: Lucy Gong
+ */
+#ifndef SENSORFUSION_HPP
+#define SENSORFUSION_HPP
+
+//Include involved sensors
+#include "IMU.hpp"
```

Comment:
Why does the caller of your module need to know about these sensors ? Isn't your module responsible for internally talking to them ?

Reply:
you're right, the SensorFusion module should be talking to them internally by calling Sensor.GetResult() - will change the header file to reflect this